core/src/display_object.rs

```diff
@@ -2066,19 +2067,12 @@ pub trait TDisplayObject<'gc>:
 
     /// Emit an `exitFrame` broadcast event.
     fn exit_frame(&self, context: &mut UpdateContext<'gc>) {
-        let exit_frame_evt = Avm2EventObject::bare_default_event(context, "exitFrame");
+        let exit_frame_evt: Avm2Object<'_> =
```

Comment:
(why was this change needed?)

Reply:
I accidentally inserted this with rust-analyer - removed